swap/types.go

```diff
@@ -41,9 +40,7 @@ type Cheque struct {
 
 // ChequeRequestMsg is sent from a creditor to the debitor to solicit a cheque
 type ChequeRequestMsg struct {
-	Peer       enode.ID // TODO: Why is it here right now? Potentially not needed as everything goes through peer
-	PubKey     []byte   // TODO: Also probably not needed
-	LastCheque *Cheque  // TODO: maybe at most just a serial number rather than full cheque
+	Beneficiary common.Address
```

Comment:
We went over this with @zelig and it appears we only need the beneficiary address for now (which will eventually be assigned to the check object).

- `Peer` is not needed as it is implicit when receiving a message.
- `LastCheque` is not needed as the receiving node will check its own last known cheque.
- `PubKey` is not needed as it was only used to calculate the beneficiary address, which can be directly given as it is now.